container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -5,7 +5,7 @@ const (
 
 	fluentBitContainerName = "fluent-bit"
 	// using debug image for now with testing toolkit (curl, netstat) eventually will move to latest regular img
-	fluentBitImage = "fluent/fluent-bit:latest-debug"
+	fluentBitImage = "fluent/fluent-bit:4.0.0-debug"
```

Comment:
Looks like Docker & k8s use different versions of fluentbit. Is this intentional? Should it be standardized?

Reply:
ah yes - i removed use of the debug image